application/controllers/cron/Keywords_stats.php

```diff
@@ -0,0 +1,32 @@
+<?php if (!defined('BASEPATH')) exit('No direct script access allowed');
+
+class Keywords_stats extends CI_Controller
+{
+	public function update_keywords_stats()
```

Comment:
I need to remember to add a line to https://github.com/LibriVox/librivox-ansible/blob/master/roles/common/files/daily-cron to actually run this. Is daily enough?